concepts/numbers/introduction.md

```diff
@@ -0,0 +1,18 @@
+# Haskell Built-in Numeric Types
+
+Integers` hold any number no matter how big, up to the limit of your machine's memory, while `Int` corresponds to the set of positive and negative integers that can be expressed in 32 or 64 bits (guaranteed at least -2^29 to 2^29).
+
+`Float` corresponds to the set of real numbers, limited by the precision of the computer.
+Operations defined on numbers usually work on one type or the other, but not mixing them.
+
+Functions to convert between `Float` and `Integer`, include `toFloat` which converts `Int\Integer` to `Float` and `round`, `floor`, `ceiling` or `truncate` which convert a `Float` to an `Int`.
```

Comment:
This is a fairly big list, most of which are functions that the student won't need. Normally, we don't list specific functions in the introduction, but only mention the package/module in which they can be found. This is to stimulate students to figure out finding these functions themselves, which will be great for their learning experience.

In this case, the above functions are all part of the prelude I think, correct? What I would suggest is to remove all but one or two functions.